### ISSUE TITLE
Reorganize hero sprites

### ISSUE THREAD

**Issue by MetalES** [MetalES/Project-Zelda#27]
Actually, as the engine expends all day, Link is having a ton of new animation, and it is not really clever to do separate files for one animation.
Merge all animations within 1 file called tunicx.png here x = the tunic number. Easier to change color, easier to set the other tunics animation up to date.

**Comment by christopho** [comment]
Actually, my experience shows that it is easier to maintain a quest with multiple PNG files for animations. For example to revert of merge changes that add or modify a specific animation.

**Comment by MetalES** [comment]
Hmmm, I mean, merge all animations that are set to be finished (items, walking, carrying, stopped), because as I modify one single thing (a frame for example), all animation set need to be modified, while this solution present a lot of advantages. (mostly time)

The only question is, how are animation loaded, is the bitmap file loaded at each animation or is it loaded just once when the animation set is loaded ?

**Comment by christopho** [comment]
They are never finished forever, and even if they were, then you cannot easily revert/merge commits that add or remove animations in other projects, or import/exports animations from/to other projects.

It is actually easier to have a PNG file with a single animation that has all colors variants in it. In a project you very often add/remove/change animations but the total number of tunic variants does not change a lot. But this is just my personal experience.

For now, the bitmap file is loaded several times, but this will be improved one day, anyway it is an implementation detail you should not consider.

**Comment by MetalES** [comment]
Hmmm, well then, I'll do it the normal way.

**Comment by Diarandor** [comment]
This is just my personal opinion:
I think it is better to keep each animation in a different png, to make them more independent each one from the others. (Sometimes I make an exception and put different animations in the same file, but not too many, to keep the image size not too big.)
Also, I usually make each variant of each animation of the hero in a different file, which makes it faster to create new variants of sprites using "copy-paste", because all variants have the same coordinates, origin, etc in each png.

**Comment by christopho** [comment]
Good point, it also allows to have identical .dat files for a lot of animations.

**Comment by MetalES** [comment]
Yeah I'm gonne use Diarandor's solution
